doc/tutorials/04-lattice_boltzmann/04-lattice_boltzmann_part2.ipynb

```diff
@@ -47,16 +47,16 @@
     "\\end{equation}\n",
     "\n",
     "It is quite difficult to prove this formula computationally with good accuracy.\n",
-    "It will need quite some computer time and a careful analysis. So please don't be\n",
+    "It will need quite some computational effort and a careful analysis. So please don't be\n",
     "too disappointed if you don't manage to do so.\n",
     "\n",
     "We want to determine the diffusion coefficient from the mean square\n",
-    "distance that a particle travels in the time $t$. For large $t$ it is\n",
-    "be proportional to the time and the diffusion coefficient occurs as \n",
+    "distance that a particle travels in the time $t$. For large $t$ the mean square distance is\n",
+    "proportional to the time and the diffusion coefficient occurs as a \n",
     "prefactor:\n",
     "\n",
     "\\begin{equation}\n",
-    "  \\frac{\\partial \\langle r^2 \\left(t\\right)\\rangle}{\\partial t} = 2 d D. \n",
+    "  \\frac{\\partial \\langle \\vec{r}^2 \\left(t\\right)\\rangle}{\\partial t} = 2 d D. \n",
```

Comment:
I think it would be clearer (and more common) to actually put down the formula for the MSD instead of
the derivative. This would also correspond more closely to how it is calculated.